src/main/java/com/example/springboottask/repository/OrderRepository.java

```diff
@@ -1,13 +1,13 @@
 package com.example.springboottask.repository;
 
-import com.example.springboottask.entity.Order;
+import com.example.springboottask.entity.OrderEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 
 import java.util.List;
 
-public interface OrderRepository extends JpaRepository<Order, Long> {
+public interface OrderRepository extends JpaRepository<OrderEntity, Long> {
 
-    @Query("select o from Order o where o.userId = ?1")
-    List<Order> findOrdersByUserId(Long userId);
+    @Query("select o from OrderEntity o where o.userId = ?1")
```

Comment:
Redundant annotation - @Query. Use the Spring Data naming method convention instead